test/__tests__/data/shopify.json

```
@@ -0,0 +1,784 @@
+[
+    {
+        "description": "Unsupported Event Type",
+        "input": {
+            "query_parameters": {
+                "topic": [
+                    "random_event"
+                ],
+                "signature": [
+                    "rudderstack"
+                ],
+                "writeKey": [
+                    "sample-write-key"
+                ]
+            }
+        },
+        "output": {
+            "error": "event type random_event not supported"
+        }
+    },
+    {
+        "description": "Identofy Call for customers create event",
```

Typo here.
Also instead of putting whole of the Shopify sent payload into test cases, can we rather have like other events' testcases too? 

We can keep a complete payload for reference sake, but there ought to be some properties or objects that are exclusive to a specific event. 

This is not a huge priority, and can be picked up post this as a test: PR adding testing testcases.

Cc @ItsSudip 
```suggestion
        "description": "Identify Call for customers create event",

---

@anantjain45823 please keep in mind adding testcases ahead or tracking the item somewhere. TCs have been super helpful in analyisis of issues